demo-app/src/pages/servers.jsx

```diff
@@ -1,20 +1,81 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { Link } from 'react-router-dom';
-import { Page, PageContent, PageHeader } from 'grommet';
-import { ReverseAnchor } from '../components';
+import {
+  Anchor,
+  Box,
+  Data,
+  DataSearch,
+  DataFilters,
+  DataSummary,
+  Grid,
+  Heading,
+  Menu,
+  Page,
+  PageContent,
+  PageHeader,
+  ResponsiveContext,
+  Toolbar,
+} from 'grommet';
+import { LinkNext } from 'grommet-icons';
+import {
+  ContentContainer,
+  ServersMap,
+  DeviceGroups,
+  RecentActivity,
+  ReverseAnchor,
+} from '../components';
+import data from '../data/servers.json';
 
 const Servers = () => {
+  const breakpoint = useContext(ResponsiveContext);
+
   return (
-    <Page fill>
-      <PageContent flex>
+    <Page kind="full" pad={{ bottom: 'xlarge' }}>
+      <PageContent>
         <PageHeader
           title="Servers"
           parent={<ReverseAnchor as={Link} label="Home" to="/" />}
         />
-        placeholder
+        <Grid
+          columns={
+            !['xsmall', 'small', 'medium'].includes(breakpoint)
+              ? ['flex', 'medium']
+              : ['auto']
+          }
+          gap="medium"
+        >
+          <ContentContainer gap="large">
+            <DataView />
+            <Box gap="medium">
+              <Box direction="row" justify="between" gap="small">
+                <Heading level={2} margin="none">
+                  Device groups
+                </Heading>
+                <Anchor label="See all" icon={<LinkNext />} reverse />
+              </Box>
+            </Box>
+            <DeviceGroups max={4} />
+          </ContentContainer>
+          <RecentActivity />
+        </Grid>
       </PageContent>
     </Page>
   );
 };
 
 export default Servers;
+
+const DataView = () => (
+  <Data data={data.servers.items}>
+    <Toolbar>
+      <DataSearch responsive />
+      <DataFilters layer />
+      <Box flex />
+      <Menu label="Actions" items={[]} kind="toolbar" />
```

Comment:
Not related to this PR but it seems like we should look at the way the Toolbar component is built. I would expect to be able to set `alignSelf="end'` on Menu instead of needing to add `<Box flex />` to achieve the desired behavior

Reply:
I think that "alignSelf" would related to where Menu would align itself from top to bottom, as opposed to left to right (since the toolbar has direction="row").

Reply:
Ah gotcha